modules/installation-creating-aws-vpc-localzone.adoc

```diff
@@ -1,35 +1,37 @@
 // Module included in the following assemblies:
 //
-// * installing/installing_aws/installing-aws-localzone.adoc
+// * installing/installing-aws-localzone.adoc (Installing a cluster on AWS with worker nodes on AWS Local Zones)
+// * installing/installing-aws-wavelength-zone.adoc (Installing a cluster on AWS with worker nodes on AWS Wavelength Zones) 
+
+ifeval::["{context}" == "installing-aws-localzone"]
+:local-zone:
+endif::[]
+ifeval::["{context}" == "installing-aws-wavelength-zone"]
+:wavelength-zone:
+endif::[]
 
 :_mod-docs-content-type: PROCEDURE
 [id="installation-creating-aws-vpc-localzone_{context}"]
 = Creating a VPC in AWS
```

Comment:
Hello @dfitzmau reviewing as requested in Slack.

The e2e steps in the section "BYO VPC for Local Zones" is not entirely correct for 4.15 ([issue 2](https://github.com/openshift/openshift-docs/pull/69537#pullrequestreview-1871083983) mentioned in my last comment): missing subnet description which impacts the full e2e.

To simplify, the ["Local Zone BYO VPC"](https://69537--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-localzone#creating-aws-local-zone-environment-existing_installing-aws-localzone) shares the same sections of Wavelength zones, except:
- [Creating a VPC carrier gateway](https://69537--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-wavelength-zone#installation-creating-aws-vpc-carrier-gw_installing-aws-wavelength-zone)
- [CloudFormation template for the VPC Carrier Gateway](https://69537--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-wavelength-zone#installation-cloudformation-vpc-carrier-gw_installing-aws-wavelength-zone)

IMO you can just use the replacing/adjusting the pieces:
- replacing `{zone-type}`
- and legend for `${ROUTE_TABLE_PUB}`

as Wavelength Zone is LGTM.

Reply:
Hi @mtulio . This should all be fixed now. I had to create three modules as an ifdef/ifndef statement issued occured with the subnet file. I was spending far too much time trying to fix the issue and it seems to me like a bug is at play.

Reply:
Not yet, I can see the intro section have been left behind.

The "[Wavelength Zones only: Creating a VPC carrier gateway](https://69537--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/aws-compute-edge-zone-tasks#installation-creating-aws-vpc-carrier-gw_aws-compute-edge-zone-tasks)" section should be **before** "[Wavelength Zones only: CloudFormation template for the VPC Carrier Gateway](https://69537--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/aws-compute-edge-zone-tasks#installation-cloudformation-vpc-carrier-gw_aws-compute-edge-zone-tasks)".